libs/agno/agno/models/meta/llama_openai.py

```diff
@@ -48,50 +48,6 @@ class LlamaOpenAI(OpenAILike):
     supports_native_structured_outputs: bool = False
     supports_json_schema_outputs: bool = True
 
-    @property
-    def request_kwargs(self) -> Dict[str, Any]:
```

Comment:
we dont need this whole function? At all?

Reply:
This was added because the behaviour had to be difficult that the default. So this isn't needed anymore